src/tests/sorting/test_sorting.f90

```diff
@@ -4,17 +4,17 @@ program test_sorting
         compiler_version
     use stdlib_kinds, only: int32, int64, dp, sp
     use stdlib_sorting
-    use stdlib_string_type, only: string_type, assignment(=), operator(>), operator(<), &
-        write(formatted)
+    use stdlib_string_type, only: string_type, assignment(=), operator(>), &
+        operator(<), write(formatted)
     use stdlib_error, only: check
 
     implicit none
 
-    integer(int32), parameter :: test_size = 2_int32**20
-    integer(int32), parameter :: char_size = 26**4
-    integer(int32), parameter :: string_size = 26**3
+    integer(int32), parameter :: test_size = 2_int32**16
```

Comment:
For flexibility, could the `16` become a parameter?

Reply:
There are two kinds of things that now have a value of 16:
1. The number of elements in the integer array are now `2**16`.
2. The number of characters in the character set used to generate the strings.
Both could be parameters, but they should be separate parameters.

Reply:
Thank you @wclodius2 . Then I think that at least one of the two values could be a parameter to clarify the code

Reply:
I have made both values parameters, committed the change and pushed it to the PR.

Reply:
Great. Thank you.